api-examples/pretrain_paired_pytorch.py

```diff
@@ -178,15 +183,16 @@ def train():
                          logger=logger, checkpoint_name=args.checkpoint)
 
     model.to(args.device)
-    loss_function = model.create_loss(args.loss)
+    if args.model_type=='dual-encoder':
```

Comment:
this casing isnt required, `create_loss()` has kwargs

Reply:
removed